docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -76,12 +76,14 @@ GitHub doesn't offer users a screen to confirm authentication. There are 2 alter
 
 2) Another way to validate linking: 
 
-> 1. Go to your profile by clicking your avatar on the top right
-> 2. Click "Settings"
-> 3. Click "Enterprises"
-> 4. Check if you're a member of "Government of British Columbia
-> 5. If you are a member, then you've successfuly authenticated
+> 1. (**Mandatory step**) Go to https://github.com/bcgov/ 
+> 2. Got to your profile by clicking your avatar on the top right
+> 3. Click "Settings"
+> 4. Click "[Sessions](https://github.com/settings/sessions)"
+> 5. Click "See more" button for the current web session
+> 6. If you're authenticated, GitHub will show details of your current session, inclusive of your device, your last login location, and when you last signed in. We've included a screenshot as an example:
```

Comment:
```suggestion
> 6. If you're authenticated, GitHub will show details of your current session, inclusive of your device, your last login location, and when you last signed in. Clicking on the three dots by the "Your session will expire in" section will display the NameID from the linked IDIR. We've included a screenshot as an example:
```

I've added the piece about the 3 dots as a question came up in platform community meetup today about how to verify which IDIR is associated.